app/src/main/java/org/oppia/android/app/utility/ClickableAreasImage.kt

```diff
@@ -116,6 +116,7 @@ class ClickableAreasImage(
         newView.isFocusable = true
         newView.isFocusableInTouchMode = true
         newView.tag = clickableArea.label
+        newView.contentDescription = clickableArea.label
```

Comment:
@aggarwalpulkit596 PTAL

This enables content description for each image region in ImageRegionSelectionInteraction
![image](https://user-images.githubusercontent.com/54740946/130150057-fc5b5715-4207-430b-b8f3-71deee6f97c6.png)

Reply:
This isn't correct. The label will give away the answer, so we introduced another property for content descriptions in image region.

However, looking at https://github.com/oppia/oppia/blob/develop/extensions/interactions/ImageClickInput/ImageClickInput.py it seems we never did add this. Per https://github.com/oppia/oppia/issues/9924 I guess we didn't finish this which introduces a problem. We unfortunately won't be able to use this method.

Can you file an issue to add support for https://github.com/oppia/oppia/issues/9924 once it's ready & add that as a TODO here to fix the reference? We probably aren't going to be able to do better than the direct indexing without content description support.

Reply:
Done
https://github.com/oppia/oppia-android/issues/3712